src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -50,7 +50,7 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
     name: "",
   });
 
-  const eventSpaceId = event?.spaceId || venue?.id;
+  const eventSpaceId = event?.spaceId || (venue?.id as SpaceId);
```

Comment:
Please remember the `undefined` otherwise TS will override the `?.` in checks.

```suggestion
  const eventSpaceId = event?.spaceId || (venue?.id as SpaceId|undefined);
```